docs/rstIconReplacements.txt

```diff
@@ -11,7 +11,7 @@
 .. |chevronUp| replace:: :raw-html:`<span class="design-system-icon"><svg viewBox="0 0 24 24" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M12 8l-6 6 1.41 1.41L12 10.83l4.59 4.58L18 14z"/></svg></span>`
 .. |classes| replace:: :raw-html:`<span class="design-system-icon"><svg role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M21.13 4H4.87a.87.87 0 00-.87.87V10h2V6h14v11h-1v2h2.13a.87.87 0 00.87-.87V4.87a.87.87 0 00-.87-.87zM8 14a2 2 0 112 2 2 2 0 01-2-2zm6 4.51V20H6v-1.49c0-1.34 2.67-2 4-2s4 .66 4 2zm0-3.35a6.36 6.36 0 00-.92.08 1.77 1.77 0 01-.23.48c1.32.49 2.48 1.37 2.48 2.79V19H18v-1.84c0-1.32-2.67-2-4-2zm0-4.49a2 2 0 00-1.68.93 3.34 3.34 0 011 2.4 3 3 0 010 .53 1.7 1.7 0 00.71.14 2 2 0 000-4zm-6.32.93A2 2 0 004 12.67a2 2 0 002 2 1.7 1.7 0 00.71-.14 3 3 0 010-.53 3.34 3.34 0 01.97-2.4zm-.76 3.64a6.36 6.36 0 00-.92-.08c-1.33 0-4 .68-4 2V19h2.67v-.49c0-1.42 1.16-2.3 2.48-2.79a1.77 1.77 0 01-.23-.48z"/></svg></span>`
 .. |clear| replace:: :raw-html:`<span class="design-system-icon"><svg viewBox="0 0 24 24" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" width="24" height="24"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg></span>`
-.. |clipboard| replace:: :raw-html:`<span class="design-system-icon"><svg viewBox="0 0 24 24" role="presentation" focusable="false" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M19 2h-4.18C14.4.84 13.3 0 12 0c-1.3 0-2.4.84-2.82 2H5c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 0c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1z"/></svg></span>`
+.. |clipboard| replace:: :raw-html:`<span class="design-system-icon"><svg viewBox="0 0 24 24" role="presentation" focusable="false" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M19 2h-4.18C14.4.84 13.3 0 12 0c-1.3 0-2.4.84-2.82 2H5c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 0c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm7 18H5V4h2v3h10V4h2v16z"/></svg></span>`
```

Comment:
@sairina this change wasn't technically part of your PR

@nucleogenesis I think we may not have committed this regenerated file after updating the clipboard icon. We should figure out how to ensure that this gets committed - I could imagine a linter rule for example

Reply:
opened a new PR to catch this in the future:

https://github.com/learningequality/kolibri-design-system/pull/138

@sairina you can revert the change to this file